core/templates/pages/preferences-page/preferences-page.component.spec.ts

```diff
@@ -266,4 +271,12 @@ describe('Preferences Controller', function() {
 
     expect(mockWindow.location.reload).not.toHaveBeenCalled();
   });
+
+  it('should call Prevent reload event service for checkbiochanged',
```

Comment:
`should add prevent reload event listener when a bio is changed`

Reply:
DONE